src/service.js

```diff
@@ -83,20 +88,27 @@ LS2Request.prototype.send = function() {
 		}
 		if(!self.subscribe) {
 			self.cancel();
+		} else if(self.ts && refs[self.ts]) {
+			delete refs[self.ts];
 		}
 	};
+	self.ts = performance.now();
```

Comment:
What are your thoughts on using the hash value of `uri` + `params` instead of `performance.now()`? That way we can keep only one ref for the same calls